src/actions/mod.rs

```diff
@@ -447,18 +447,34 @@ impl ActionHandler {
                 return;
             }
         };
-        let config = self.fmt_config.lock().unwrap();
+
+        let range_whole_file = ls_util::range_from_vfs_file(&self.vfs, path);
+        let range = match selection {
+            Some(r) => r,
+            None => range_whole_file,
+        };
+        let range_rls = ls_util::range_to_rls(range).one_indexed();
+        let mut config = self.fmt_config.lock().unwrap().get_rustfmt_config().clone();
+
+        let file_lines = format!(r#"[{{
+            "file": "stdin",
+            "range": [{}, {}]
+        }}]"#, range_rls.row_start.0, range_rls.row_end.0);
+        config.set().file_lines(file_lines.parse().unwrap());
```

Comment:
COuld we move all of this stuff inside the match/Some branch (and the decl of config before the match)? I'd like to avoid using the file_lines change if no range was given.